docs/developer_guide.md

```diff
@@ -23,15 +23,16 @@ Install latest VS2017 (15.9 or later) from here: http://visualstudio.com/downloa
 
 While WinUI is designed to work against many versions of Windows, you will need 
 a fairly recent SDK in order to build WinUI. It's required that you install the 
-16299, 17134 and 17763 SDKs. You can download these via Visual Studio (check 
+16299, 17134, 17763 and 18362 SDKs. You can download these via Visual Studio (check 
 all the boxes when prompted), or you can manually download them from here: 
 https://developer.microsoft.com/en-US/windows/downloads/windows-10-sdk
 
-You will also need to install The Windows 10 Insider SDK 18312. The easiest way 
+<!-- 
+You will also need to install The Windows 10 Insider SDK 18323. The easiest way 
 to install this is to run the Install-WindowsSdkISO.ps1 script from this repo in
 an Administrator Powershell window:
 
- `.\build\Install-WindowsSdkISO.ps1 18323`
+ `.\build\Install-WindowsSdkISO.ps1 18323` -->
```

Comment:
```suggestion
 `.\build\Install-WindowsSdkISO.ps1 18362` -->
```

Reply:
Note that this is commented out. It's not needed anymore but I wanted to leave it for the next person who wants to retarget to insider.